msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1121,6 +1138,22 @@ private void setupTelemetry(@NonNull final Context context,
                 .build();
     }
 
+    @Override
+    public String getBrokerRt(@NonNull final String accountObjectId) throws MsalClientException {
```

Comment:
Why does this need to be part of the public API of MSAL?  Is there an objection to this being accessed via an internal or private method?

Reply:
CP and AuthApp has to call it. Currently they're using ADAL to obtain Broker AT/RT, then pass it into WPJ's Join() operation as parameters.

Reply:
@rpdome @shoatman We could do like we did for TSL APIs and hide it away.... There's an `ITokenShare` API that `PublicClientApplication` needs to be cast to before those methods can be used. I think exposing this method here _and_ not allowing non-broker apps to call it is going to be incredible frustrating for those developers who have requested we expose RT in the past

Reply:
That's what I'm aiming for. IBrokerRefreshTokenAccessor = ITokenShare.

Reply:
@rpdome Sounds just fine to me then 👍

Reply:
@iambmelt - One of the goals of our libraries is not to expose refresh tokens.  Anyone can access the refresh token if they really want to... but we don't want to make it easy.  Also, what will we do when we enable refresh token PoP?  Are we planning to allow them to export the private key as well?

Reply:
@rpdome - I think we should have a quick meeting to discuss this in more detail.

Reply:
@rpdome @shoatman Agree, let's sync on these points

Reply:
Updated as synced.